frontend/src/Stats.js

```diff
@@ -11,28 +14,52 @@ import {
   CartesianGrid,
   ResponsiveContainer,
 } from 'recharts'
+import theme from './theme'
 
 class Chart extends PureComponent {
   render() {
     return (
-      <div style={{ width: '100%', height: 300 }}>
-        <ResponsiveContainer>
-          <AreaChart
-            data={this.props.data}
-            margin={{
-              top: 0,
-              right: 0,
-              left: -25,
-              bottom: 0,
-            }}
-          >
-            <CartesianGrid strokeDasharray="2 2" />
-            <XAxis dataKey="date" />
-            <YAxis dataKey="total" />
-            <Area type="monotone" dataKey="total" stroke="#000" fill="#999" />
-          </AreaChart>
-        </ResponsiveContainer>
-      </div>
+      <I18n>
+        {({ i18n }) => (
+          <React.Fragment>
+            <H3 mt={6} textAlign="center">
+              <Trans>Reports similar to yours</Trans>
+            </H3>
+            <div style={{ width: '100%', height: 300 }}>
+              <ResponsiveContainer>
+                <AreaChart
+                  data={this.props.data}
+                  margin={{
+                    top: 0,
+                    right: 0,
+                    left: -20,
+                    bottom: 0,
+                  }}
+                >
+                  <CartesianGrid strokeDasharray="2 2" />
+                  <XAxis fontFamily={theme.fontSans} dataKey="date" />
```

Comment:
Using the `ThemeProvider` didn't seem to work here, unless I was doing it the wrong way.

Reply:
Can we make these changes in French too?

Title EN: Reports similar to yours
Title FR: Rapports semblables au vôtre

Y-axis EN: Reports per day
Y-axis FR: Rapports par jour

Reply:
Right now the French (just from DeepL) is:
```
"Reports per day": "Rapports par jour",
"Reports similar to yours": "Des rapports similaires aux vôtres",
```
Can change the French title as you suggest 👍

Reply:
<img width="1132" alt="image" src="https://user-images.githubusercontent.com/8228248/56037918-8c7d9600-5cfe-11e9-8d6d-cc7185f5d36a.png">